conf/pvers.c

```diff
@@ -810,6 +810,31 @@ static int conf_pver_virtual_create(const struct m0_fid *fid,
 	return rc;
 }
 
+/**
+ * Builds a vector consisting of failed/offline objects in the conf tree of a
+ * pool version. Repaired devices are not considered to be failed.
+ */
+static int conf_pver_failed_recd_build(struct m0_conf_obj *obj, void *args)
```

Comment:
How about renaming function name "conf_pver_failed_recd_build" to "conf_pver_nonhealty_recd_build"?

Reply:
Done.